components/badge.json

```diff
@@ -4,6 +4,7 @@
     "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows%2BWeb?node-id=4486%3A786",
     "background": "@theme-background-accent-normal",
     "text": "@theme-common-text-white",
-    "notification-indicator": "@theme-text-accent-normal"
+    "notification-indicator": "@theme-text-accent-normal",
+    "recording-indicator": "@theme-indicator-attention"
```

Comment:
why is this on the badge component file?

Reply:
terms indicator and badges will get used interchangeably from time to time. It really just means that something has changed status. Simplest way it can be described I think